homeassistant/components/sensibo/strings.json

```diff
@@ -40,9 +62,83 @@
+        }
+      },
+      "sensibo_sensor": {
+        "state_attributes": {
+          "fanlevel": {
+            "name": "Fan level",
+            "state": {
+              "auto": "Auto",
+              "high": "High",
+              "low": "Low",
+              "medium": "Medium",
+              "medium_high": "Medium high",
+              "quiet": "Quiet"
+            }
+          },
+          "horizontalswing": {
+            "name": "Horizontal swing",
+            "state": {
+              "stopped": "[%key:component::sensibo::entity::select::horizontalswing::state::stopped%]",
+              "fixedleft": "[%key:component::sensibo::entity::select::horizontalswing::state::fixedleft%]",
+              "fixedcenterleft": "[%key:component::sensibo::entity::select::horizontalswing::state::fixedcenterleft%]",
+              "fixedcenter": "[%key:component::sensibo::entity::select::horizontalswing::state::fixedcenter%]",
+              "fixedcenterright": "[%key:component::sensibo::entity::select::horizontalswing::state::fixedcenterright%]",
+              "fixedright": "[%key:component::sensibo::entity::select::horizontalswing::state::fixedright%]",
+              "fixedleftright": "[%key:component::sensibo::entity::select::horizontalswing::state::fixedleftright%]",
+              "rangecenter": "[%key:component::sensibo::entity::select::horizontalswing::state::rangecenter%]",
+              "rangefull": "[%key:component::sensibo::entity::select::horizontalswing::state::rangefull%]"
+            }
+          },
+          "light": {
+            "name": "Light",
+            "state": {
+              "on": "[%key:component::sensibo::entity::select::light::state::on%]",
+              "dim": "[%key:component::sensibo::entity::select::light::state::dim%]",
+              "off": "[%key:component::sensibo::entity::select::light::state::off%]"
+            }
+          },
+          "mode": {
+            "name": "Mode"
+          },
+          "on": {
+            "name": "On",
+            "state": {
+              "true": "On",
+              "false": "Off"
+            }
+          },
+          "swing": {
+            "name": "Swing",
+            "state": {
+              "both": "Both",
+              "fixedbottom": "Fixed bottom",
+              "fixedmiddle": "Fixed middle",
+              "fixedmiddlebottom": "Fixed middle bottom",
+              "fixedmiddletop": "Fixed middle top",
+              "fixedtop": "Fixed top",
+              "horizontal": "Horizontal",
+              "rangebottom": "Range bottom",
+              "rangefull": "Range full",
+              "rangemiddle": "Range middle",
+              "rangetop": "Range top",
+              "stopped": "Stopped"
+            }
+          },
+          "targettemperature": {
+            "name": "Temperature target"
+          },
+          "temperatureunit": {
```

Comment:
Why do we have this state attribute ? Can’t we already see the unit .

Reply:
The full config for the specific Climate React setting are in the attributes including the temperature unit.
Could probably get rid off it but most likely simpler to leave it there
![image](https://github.com/home-assistant/core/assets/62932417/25de0cf4-cdd8-4cd1-ad14-5607d0d60768)